### PR TITLE
implemented task and tests

### PR DESCRIPTION
**Pull request by ivadimko**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
-# Boilerplate for JS tasks
+# The "Bulls & Cows" game
 
 **Read the guideline before start**
 
 [Guideline](https://github.com/mate-academy/js_task-guideline/blob/master/README.md)
+
+## Rules
+
+- computed makes a number of 4 different digits
```
Comment (roman-mate) [inline readme.md]: computed -> compter randomly generates a number of ..

File: src/startGame.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    const [bulls, cows] = result;
+
+    answer = `The result is ${bulls} bull(s) and ${cows} cow(s)`;
+  }
+
+  console.log(
+    `Computer generated ${generatedNumber}, you entered ${enteredNumber}.
+    ----
+    ${answer}
+    `
```
Comment (roman-mate) [inline src/startGame.js]: suggestion: move closing ` and );  to the line above?

File: readme.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+- `bull` - guessed digit is on its place (index)
+- `cow` - guessed digit presents in the number but place (index) is wrong
+
+## Example
+Computer makes `1234`, user prints `1345`. The result is one `bull` (guessed
+digit `1` is on it's place) and 2 `cows` (digits `3` and `4` are present but on
+wrong places). 
+
+## Additional scripts
+`npm run play` - run game
```
Comment (roman-mate) [inline readme.md]: Like it!

File: readme.md [hunk 4]
```diff
@@ -1,5 +1,21 @@
...
 
 [Guideline](https://github.com/mate-academy/js_task-guideline/blob/master/README.md)
+
+## Rules
+
+- computer randomly generates a number of 4 different digits
+- player enters a number of 4 different digits too
+- computer compares given numbers and shows a result of 'bulls' and 'cows'
+- `bull` - guessed digit is on its place (index)
+- `cow` - guessed digit presents in the number but place (index) is wrong
```
Comment (mgrinko) [inline readme.md]: guessed digit  `exists` in the number but `the` place

File: src/bullsAndCows.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+/**
+ * Implement bullsAndCows function:
+ *
+ * Function takes 2 strings: generatedNumber (generated by computer) and
+ * enteredNumber (entered by player) and returns count of bulls and cows
+ * according to rules (see readme.md). Each parameter should be at most 4 digits
```
Comment (mgrinko) [inline src/bullsAndCows.js]: exactly 4 digits

File: src/bullsAndCows.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+ * long.
+ *
+ * Function returns result as array of numbers where result[0] is a count of
+ * bulls and result[1] is a count of cows.
```
Comment (mgrinko) [inline src/bullsAndCows.js]: And it is a task requiring Arrays but not only strings and numbers

Comment (mgrinko) [reply src/bullsAndCows.js]: I would prefer to have { bulls: 1, cows: 2 } object instead

**Comment by ivadimko** [comment]
@mgrinko please check now
https://github.com/mate-academy/js_bulls-and-cows/pull/1/commits/98c5f9598580ebdbd83a6df752f197f3340cd7c7

File: src/bullsAndCows.js (hunk 6 above)
Comment (roman-mate) [reply src/bullsAndCows.js]: good point!

**Pull request merged**